src/pretalx/orga/forms/cfp.py

```diff
@@ -59,6 +49,13 @@ class CfPSettingsForm(
         ),
         required=False,
     )
+    show_duration_in_submission_type = forms.BooleanField(
```

Comment:
I think the discussion in #1976 involved **not** adding a separate setting, right? That was the reason why the feature request was accepted, after all. As far as I can tell, you don’t use this field for determining if the duration should be shown, so I’m confused as to why you added it. Could you please explain your reasoning?